src/apprt/gtk/i18n.zig

```diff
@@ -0,0 +1,37 @@
+//! I18n support for the GTK frontend based on gettext/libintl
+//!
+//! This is normally built into the C standard library for the *vast* majority
+//! of users who use glibc, but for musl users we fall back to the `gettext-tiny`
+//! stub implementation which provides all of the necessary interfaces.
+//! Musl users who do want to use localization should know what they need to do.
```

Comment:
I don't think Ghostty even builds for musl right now anyways but this is a good thought regardless to not give us any more debt to work on there. (I think the debt is mainly we have other deps that don't play nice with musl)

Reply:
Yeah honestly like... I don't even know how people who use musl even manage to coax GTK & Friends to work on it, but if they do manage to do it then I *presume* they have enough know-how to get localization working